docs.md

```diff
@@ -1235,10 +1235,11 @@ end)
 
 The `options` table accepts the following fields:
 
-  - `options.args` - Command line arguments as a list of string. The first
-  string should be the path to the program. On Windows, this uses CreateProcess
-  which concatenates the arguments into a string. This can cause some strange
-  errors. (See `options.verbatim` below for Windows.)
+  - `options.args` - Command line arguments as a list of strings. The first
+  string should *not* be the path to the program, since that is already
+  provided via `path`. On Windows, this uses CreateProcess which concatenates
```

Comment:
provided via `path` by `luv`, this is slightly different with `libuv`.

Reply:
I think it's okay to leave this out because these are the `luv` docs, so it should be implied.